doxygen/examples/intro_SWMR.html

```diff
@@ -0,0 +1,101 @@
+<html>
+  <head>
+    <title>Introduction to Single-Writer_Multiple-Reader (SWMR)</title>
+
+<p>*** UNDER CONSTRUCTION ***</p>
```

Comment:
I feel like we should remove the UNDER CONSTRUCTION lines

Reply:
Agree.

Reply:
Yes, I will.  It's just that I haven't been able to test the images until it can be merged.

Reply:
Done